VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -208,12 +211,19 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
                     if let pipController = self.pictureInPictureController as? AVPictureInPictureController {
                         return pipController
                     } else {
-                        guard
-                            let layer = videoView?.playerLayer,
-                            props.pictureInPictureActive else { return nil }
+                        guard let layer = videoView?.playerLayer else { return nil }
                         let pipController = AVPictureInPictureController(playerLayer: layer)
```

Comment:
Maybe just have `guard let pipController = AVPictureInPictureController(playerLayer: layer)`.
Then you don't need `if let` at the bottom.